src/modules/sub-items/sub.items.module.js

```diff
@@ -344,13 +380,141 @@ export default class SubItemsModule extends Component {
 
         if (movedItems.length) {
             const message = Translator.trans(
-                /*@Desc("The selected content item(s) have been sent to {{ locationLink }}")*/ 'bulk_move.success.message',
-                {},
+                /*@Desc("The selected content item(s) have been sent to {{ locationLink }}")*/
+                'bulk_move.success.message', {}, 'sub_items'
+            );
+            const rawPlaceholdersMap = {
+                locationLink: Translator.trans(
+                    /*@Desc("<u><a href='%locationHref%'>%locationName%</a></u>")*/
+                    'bulk_action.success.link_to_location',
+                    {
+                        locationName: eZ.helpers.text.escapeHTML(location.ContentInfo.Content.Name),
+                        locationHref: this.props.generateLink(location.id),
+                    },
+                    'sub_items'
+                ),
+            };
+
+            window.eZ.helpers.notification.showSuccessNotification(message, () => {}, rawPlaceholdersMap);
+        }
+    }
+
+    afterBulkHide(successItems, failedItems) {
+        this.deselectAllItems();
+        this.discardActivePageItems();
+
+        this.toggleBulkOperationStatusState(false);
+
+        if (failedItems.length) {
+            const modalTableTitle = Translator.trans(
+                /*@Desc("Content items cannot be hidden (%itemsCount%)")*/
+                'bulk_hide.error.modal.table_title', {itemsCount: failedItems.length}, 'sub_items'
+            );
+            const notificationMessage = Translator.trans(
+                /*@Desc("%failedCount% of the %totalCount% selected item(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.")*/
```

Comment:
```suggestion
                /*@Desc("%failedCount% of the %totalCount% selected item(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator to obtain permissions.")*/
```